src/react/ISV/components/__tests__/ISVApplyActions.test.tsx

```diff
@@ -0,0 +1,303 @@
+import { render, screen, fireEvent } from '@testing-library/react';
```

Comment:
For this test, I think we mock a bit too many things making the unit test too tightly coupled with the current implementation. I'm ok to approve this first iteration and review this after